spring-test/src/main/java/org/springframework/test/util/JsonExpectationsHelper.java

```diff
@@ -36,10 +37,11 @@
 	 *
 	 * @param expected the expected JSON content
 	 * @param actual the actual JSON content
+	 * @param compareMode the comparing mode
 	 * @since 4.1
 	 */
-	public void assertJsonEqual(String expected, String actual) throws Exception {
-		JSONAssert.assertEquals(expected, actual, false);
+	public void assertJsonEqual(String expected, String actual, JSONCompareMode compareMode) throws Exception {
+		JSONAssert.assertEquals(expected, actual, compareMode);
```

Comment:
These are breaking changes.

If you want to introduce something like this, you will have to introduce overloaded methods that accept the new method argument, keeping the existing signatures in tact.